### ISSUE TITLE
Improve accessibility of the InfoTooltip component

### ISSUE THREAD

**Issue by pnavarrc** [covid-projections/act-now-packages#211]
_work in progress_

A keyboard-only user should be able to open the tooltip and operate any interactive (button, link) elements inside the tooltip. 

- https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/
- https://www.w3.org/WAI/ARIA/apg/patterns/dialogmodal/
